Cargo.toml

```diff
@@ -19,6 +19,6 @@ panic = 'abort'
 codegen-units = 1
 
 [patch.crates-io]
-hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "576683b9f2865f1118c309017ff36e01f84420c9" }
+hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "9b4c1bdbe0517e3a7a8f285d6048a37d472ba3bc" }
```

Comment:
I confirmed that this commit corresponds to https://github.com/jjyr/hdwallet/pull/16. I also confirmed that we have not released a `zcashd` version (other than `4.7.0-rc1`) that pins `576683b9`, so it's okay if that commit disappears. We need to ensure that `9b4c1bdb` **DOES NOT** disappear, as a public `zcashd` source release will depend on it.